include/staff/ticket-view.inc.php

```diff
@@ -453,7 +453,7 @@ class="icon-folder-close-alt icon-fixed-width"></i> %s</a></li>',
                     <?php echo Format::htmlchars($user->getOrganization()->getName()); ?>
                         <a href="tickets.php?<?php echo Http::build_query(array(
                             'status'=>'open', 'a'=>'search', 'orgid'=> $user->getOrgId()
-                        )); ?>" title="<?php echo __('Related Tickets'); ?>"
+                        )); ?>" title="<?php echo __('Organization Related Tickets'); ?>"
```

Comment:
This can be broken into two parts: `Organization` and `Related Tickets` using `sprintf()`